scanapi/__init__.py

```diff
@@ -49,4 +57,4 @@ def scan(spec_path, docs_path, log_level):
         return
 
     responses = RequestsMaker(api_tree.leaves).make_all()
-    DocsWriter(docs_path).write(responses)
+    Reporter(docs_path, reporter, template).write(responses)
```

Comment:
Now only a single concrete class that knows what jinja template should be selected